adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala

```diff
@@ -64,6 +64,25 @@ private[adam] class GenotypeArraySerializer extends IntervalArraySerializer[Refe
   }
 }
 
+object GenotypeRDD extends Serializable {
+
+  /**
+   * Builds a GenotypeRDD without a partition map.
+   *
+   * @param rdd The underlying RDD.
+   * @param sequences The sequence dictionary for the RDD.
+   * @param samples The samples for the RDD.
+   * @param headerLines The header lines for the RDD.
+   * @return A new GenotypeRDD.
+   */
+  def apply(rdd: RDD[Genotype],
+            sequences: SequenceDictionary,
+            @transient samples: Seq[Sample],
```

Comment:
Ok, thanks. I didn't know if it was needed in the apply fields of the factory objects.